toqito/channel_ops/kraus_to_choi.py

```diff
@@ -15,7 +15,9 @@ def kraus_to_choi(kraus_ops: list[list[np.ndarray]], sys: int = 2) -> np.ndarray
     The Choi matrix of the list of Kraus operators, :code:`kraus_ops`. The default convention is
     that the Choi matrix is the result of applying the map to the second subsystem of the
     standard maximally entangled (unnormalized) state. The Kraus operators are expected to be
-    input as a list of numpy arrays.
+    input as a list of numpy arrays (i.e. [[:math:`A_1`, :math:`B_1`],...,[:math:`A_n`, :math:`B_n`]]).
```

Comment:
Since we are referring to `numpy` arrays here, we likely want to use :code: over :math:, that is:

```suggestion
    input as a list of numpy arrays (i.e. [[:code:`A_1`, :code:`B_1`],...,[:code:`A_n`, :code:`B_n`]]).
```